krank failing to build on stackage

`krank` is failing to build on stackage nightly (GHC 8.10.1) and we have disabled it. When you have a fix could you droip us a line so we can reenable it. Cheers.
```
--  While building package krank-0.2.1 using:
      /var/stackage/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_3.2.0.0_ghc-8.10.1 --builddir=.stack-work/dist/x86_64-linux/Cabal-3.2.0.0 build lib:krank exe:krank --ghc-options ""
    Process exited with code: ExitFailure 1
    Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/krank-0.2.1.log

    Configuring krank-0.2.1...
    Preprocessing library for krank-0.2.1..
    Building library for krank-0.2.1..
    [1 of 9] Compiling Krank.Types
    [2 of 9] Compiling Krank.Checkers.Ignore

    /var/stackage/work/unpack-dir/unpacked/krank-0.2.1-c46cf6652b9f49e0bd80ac830190ff14b511644f2f1a394cc28189c138c528e4/src/Krank/Checkers/Ignore.hs:73:19: error:
        Ambiguous occurrence ‘foldl’
        It could refer to
           either ‘Prelude.foldl’,
                  imported from ‘Prelude’ at src/Krank/Checkers/Ignore.hs:4:8-28
                  (and originally defined in ‘Data.Foldable’)
               or ‘HashM.foldl’,
                  imported from ‘Data.HashMap.Strict’ at src/Krank/Checkers/Ignore.hs:12:1-35
                  (and originally defined in ‘unordered-containers-0.2.11.0:Data.HashMap.Base’)
       |
    73 |     ignoreIndex = foldl f HashM.empty ignoreCommands
       |                   ^^^^^
```

--- a/src/Krank/Checkers/Ignore.hs
+++ b/src/Krank/Checkers/Ignore.hs
@@ -9,7 +9,7 @@ where
 
 import qualified Data.ByteString.Char8 as ByteString
 import Data.ByteString.Char8 (ByteString)
-import Data.HashMap.Strict as HashM
+import qualified Data.HashMap.Strict as HashM
 import qualified Data.List as DataL
 import Krank.Types
 import PyF (fmt)

